tests/lib/rules/jsx-no-literals.js

```diff
@@ -44,6 +44,20 @@ ruleTester.run('jsx-no-literals', rule, {
 
   valid: [
     {
+      code: `
+        class Comp1 extends Component {
+          render() {
+            return (
+              <div>
+                <button type="button" ></button>
+              </div>
+            );
+          }
+        }
+      `,
+      options: [{noStrings: true, allowedStrings: ['button', 'submit']}],
+      parser: parsers.BABEL_ESLINT
```

Comment:
let's also add the same test case for each of the other parsers (default, and both typescript-eslints)

Reply:
Can you please clarify?

Reply:
This test case works for babel-eslint; but the same test case should pass on the default parser, and on the old typescript-eslint, and on the new typescript eslint parser.

Reply:
Thanks, added!